spaceship.py

```diff
@@ -0,0 +1,137 @@
+class Spaceship:
+
+    def __init__(self):
+        self.thirst = 50
+        self.work = 50
+        self.horde = 50
+        self.hunger = 50
+        self.receipts = 50
+        self.stable = True
+        self.total = 0
+
+    def drink(self):
+        self.thirst -= 10
+        self.work += 2
+
+    def deliver(self):
+        self.work += 8
+        self.receipts += 4
+
+    def steal(self):
+        self.horde += 10
+        self.work += 7
+
+    def eat(self):
+        self.hunger -= 10
+        self.work += 2
+
+    def account(self):
+        self.receipts -= 10
+        self.horde -= 10
+
+    def thirst_level(self):
+        if self.thirst < 50:
+            self.total += 7
+        elif self.thirst >= 60:
+            self.total -= 7
+
+    def horde_level(self):
+        if self.horde > 80:
+            self.total += 5
+        elif self.horde <= 80:
+            self.total -= 1
+
+    def work_level(self):
+        if self.work < 70:
+            self.total += 1
+        elif self.work >= 70:
+            self.total -= 1
+
+    def hunger_level(self):
+        if self.hunger > 60:
+            self.total -= 1
+        elif self.hunger <= 60:
+            self.total -= 1
+
+    def receipt_level(self):
+        if self.receipts > 70:
+            self.total += 3
+        elif self.receipts <= 70:
+            self.total -= 3
+
+    def total_score(self):
+        self.thirst_level
+        self.horde_level
+        self.work_level
+        self.receipt_level
+        self.hunger_level
+        return self.total
+
+    def stable(self):
+        if any( [self.thirst > 99, self.work > 99, self.horde < 1, self.hunger > 99, self.receipts < 1] ):
```

Comment:
Interesting use of the `any`! As a nitpick, I think this would use more computing power than checking individual conditionals, just something to keep in mind in the future